src/stats/stats_reader.cpp

```diff
@@ -190,6 +190,7 @@ void update_delta_stat(socket_stats_t* p_curr_stat, socket_stats_t* p_prev_stat)
 	p_prev_stat->counters.n_rx_poll_miss = (p_curr_stat->counters.n_rx_poll_miss - p_prev_stat->counters.n_rx_poll_miss) / delay;
 	p_prev_stat->counters.n_rx_poll_hit = (p_curr_stat->counters.n_rx_poll_hit - p_prev_stat->counters.n_rx_poll_hit) / delay;
 	p_prev_stat->n_rx_ready_byte_count = p_curr_stat->n_rx_ready_byte_count;
+	p_prev_stat->n_tx_ready_byte_count = p_curr_stat->n_tx_ready_byte_count;
```

Comment:
@OphirMunk `n_tx_ready_byte_count` is consistent with `n_rx_ready_byte_count`. I think that it is good approach to name rx/tx members identically.
Naming the same statistic variables in different ways for rx and tx can lead to confuse.
What do you think?